src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -945,6 +950,9 @@ public void NotifyAboutClusterTopologyAndConnectivityChanges()
 
         private void OnTopologyChangeInternal(ClusterTopology topology, Dictionary<string, NodeStatus> status = null)
         {
+            if (_lastClusterTopologyIndex < topology.Etag)
```

Comment:
@garayx do we need to read it during ServerStore initialization?

Reply:
done